recipes/mixsimulator/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "mixsimulator" %}
+{% set version = "0.2.9.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mixsimulator-{{ version }}.tar.gz
+  sha256: 0990c51202902c5e3a7bfa747b7e5869877738e21306d8e86763d7513214671c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - mixsimulator
+    - mixsimulator.centrals
+
+about:
+  home: https://github.com/Foloso/MixSimulator
+  summary: Python application with nevergrad optimization model for calculating and simulating the least cost of an energy Mix under constraints.
+  license_file: LICENSE.txt
```

Comment:
I opened a PR upstream to deal with having to manually package the license file (https://github.com/Foloso/MixSimulator/pull/5). 

Please check this upon the next release to see if you can drop manually packaging the license file.